src/actions/index.js

```diff
@@ -43,28 +38,26 @@ export const login = values => (dispatch) => {
     });
 };
 
-export const rsvpResponse = (user, form, push) => (dispatch) => {
-  // rsvpRef
-  //   .doc(user.uid)
-  //   .set(form)
-  //   .then(() => {
-  //     dispatch({ type: types.UPDATE_RSVP, rsvpVal: true });
-  //     push('/dashboard');
-  //   });
+export const rsvpResponse = () => (dispatch) => {
+  dispatch({ type: types.UPDATE_RSVP, rsvpVal: true });
```

Comment:
Don't include rsvpResponse changes in the Forgot Password PR

Should be in a separate PR, or bundled together and put as a different name. Since Jumer assigned just the Forgot Password, I would leave this out for now.